src/main/java/com/nvidia/spark/rapids/jni/BooleanUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.*;
+
+public class BooleanUtils {
+
+  /**
+   * Get the number of false values in a boolean/int8/uint8 column.
+   */
+  public static long falseCount(ColumnView input) {
+    if (input.getRowCount() == 0) {
+      return 0L;
+    }
+
+    return falseCount(input.getNativeView());
+  }
+
+  /**
+   * Get the number of true values in a boolean/int8/uint8 column.
+   */
+  public static long trueCount(ColumnView input) {
+    return input.getRowCount() - falseCount(input);
+  }
+
+  private static native long falseCount(long handle) throws CudfException;
```

Comment:
I don't think we need any of them in JNI. These can be computed on the fly in the plugin:
 * For `trueCount`, we can compute it as `sum(DType.INT32)`.
 * `falseCount = rowCount - trueCount()`.